### ISSUE TITLE
Model Predict Hallucination Issues

### ISSUE THREAD

**Issue by shengyuwoo** [linkangheng/PR1#3]
How to address the hallucination problem in predictions? Have you conducted comparative experiments on prediction-induced hallucinations? For example, when performing object detection predictions on an image containing non-existent objects, does the model still output bounding boxes for those non-existent targets?

If you have effective methods for reducing hallucination in model predictions, could you share them?

Thank you for your work.

**Comment by linkangheng** [comment]
Thanks for your question. We indeed observed the hallucination phenomena in object detection tasks including:

1. **Misclassification** of objects with incorrect labels
2. **Missed detections** of clearly present objects
3. **False positives** in empty image regions
4. **Phantom bounding boxes** around non-existent objects

And designing appropriate reward functions can effectively mitigate this problem. As detailed in Table 6 of our paper, through implementing multiple reward functions that balances detection accuracy with penalties, we observed significant reduction in hallucination instances.

**Comment by shengyuwoo** [comment]
How did you design the labeling format for the target detection training set? Can you give a few examples of labels for the training dataset?
